### PR TITLE
ft/test-config

### PR DESCRIPTION
**Pull request by bibi-ic**
- [x] add test for `config` package
- [x] add feature `coverage` for MAKE phony

### CODE REVIEW

File: config/config_test.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+		assert.Equal(t, expectedString, config.DBDriver)
+		assert.Equal(t, expectedString, config.DBSource)
+		assert.Equal(t, expectedString, config.TokenSemmetricKey)
+		assert.Equal(t, expectedString, config.ServerAddress)
+		assert.Equal(t, expectedTime, config.AccessTokenDuration)
+		assert.Equal(t, expectedTime, config.RefreshTokenDuration)
+	})
+
+	t.Run("Config file set", func(t *testing.T) {
```
Comment (9bany) [inline config/config_test.go:59]: Clear this file with testcases list

**Review by 9bany — CHANGES_REQUESTED** [review]
Please, improve your testcases

File: config/config_test.go (hunk 1 above)
Comment (bibi-ic) [reply config/config_test.go]: resolved @9bany

File: config/config_test.go [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+		tc := testCases[i]
+		t.Run(tc.name, func(t *testing.T) {
+			fs := afero.NewMemMapFs()
+
+			absFilePath, err := filepath.Abs("./testsuite/apptest.env")
+			if err != nil {
+				t.Fatal(err)
+			}
+
+			file, err := fs.Create(absFilePath)
```
Comment (9bany) [inline config/config_test.go:74]: please ! remove your env file after test running !

**Review by 9bany — CHANGES_REQUESTED** [review]

File: config/config_test.go (hunk 2 above)
Comment (bibi-ic) [reply config/config_test.go:74]: @9bany this is virtual env from afero package Sifu.

So is does not create any on local test

https://github.com/spf13/afero#using-afero-for-testing

File: config/config_test.go [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+			err = viper.ReadInConfig()
+			if err != nil {
+				if _, ok := err.(viper.ConfigFileNotFoundError); ok {
+					t.Fatal("File Not Found")
+				} else {
+					t.Fatal("Config File was found but another error was produced")
+				}
+			}
+
+			var config Config
```
Comment (9bany) [inline config/config_test.go:93]: move from line 93-101 to a function

**Review by 9bany — CHANGES_REQUESTED** [review]

File: config/config_test.go (hunk 3 above)
Comment (bibi-ic) [reply config/config_test.go]: @9bany resolved

**Pull request merged**